External/FEXCore/Source/Interface/Config/Config.json.in

```diff
@@ -312,6 +312,14 @@
           "\tblocks: Will enable disassembly of the translated instruction code blocks",
           "\tstats: Will print stats when disassembling the code"
         ]
+      },
+      "ForceSVEWidth": {
+        "Type": "uint32",
+        "Default": "0",
+        "Desc": [
+          "Allows overriding the SVE width in the vixl simulator.",
+          "Useful as a debugging feature."
+        ]
```

Comment:
If we ever get to test on some 256-bit hardware, this could maybe also use `prctl(PR_SVE_SET_VL, ...)` to configure the hardware vector length on Linux (provided it's less than the supported vector length).